third_party/move/move-compiler/tests/move_check/parser/invalid_call_lhs_complex_expression.exp

```diff
@@ -1,9 +1,18 @@
-error[E01002]: unexpected token
-  ┌─ tests/move_check/parser/invalid_call_lhs_complex_expression.move:3:29
+error[E02004]: invalid 'module' declaration
```

Comment:
Is this update on error message legit?

Reply:
Yes.  The error was previously shadowed by the unsupported calls being caught in the parser.  Now V1 proceeds through to a later phase where the module declaration error also shows up.